File: introducao_programacao_estruturada/teoria/22.08/soma.py
# Digite um programa para ler valores e somar até digitar 0

print("\nPrograma para ler e somar números.\n")

soma: int = 0

while True:
  numero = int(input("Informe um número inteiro ou digite 0 para sair:"))
  
  if numero == 0:
    break
  
  soma += numero
  print(f"Soma: {soma}")
  

File: logica_programacao_algoritmos/lista4/soma.pas
// Elaborar um programa que efetue a leitura de 15 valores numéricos inteiros e 
// no final apresente o total do somatório desses números.

Algoritmo "SOMA 15"
// Disciplina   : [Lógica de Programação e Algoritmos]
// Professor   : Marcelo Akira
// Descrição   : Lista 4 - Exercício 2
// Autor(a)    : Ana Espejo
// Data atual  : 08/10/2024
Var
  numero, contador, soma: inteiro

Inicio
  escreval("Programa soma 15 números")
  para contador de 1 ate 15 faca
    escreva("Informe o ", contador, "° número: ")
    leia(numero)

    soma <- soma + numero;
  fimpara
  escreva("A soma dos números digitados é: ", soma)
Fimalgoritmo
